opentech/apply/stream_forms/blocks.py

```diff
@@ -384,6 +384,10 @@ class FormFieldsBlock(StreamBlock):
     image = ImageFieldBlock(group=_('Fields'))
     file = FileFieldBlock(group=_('Fields'))
     multi_file = MultiFileFieldBlock(group=_('Fields'))
+    group = BooleanBlock(
+        group=_('Custom'), label=_('Group fields'), template='stream_forms/render_group_field.html',
+        icon='group', help_text='This field will not render on the page. Only helps to group fields.'
+    )
```

Comment:
This needs to be checked I discovered, maybe have it be checked by default in that case.

Reply:
Created a new block instead of using the boolean block for toggle effect.